dask_expr/_expr.py

```diff
@@ -1499,6 +1499,15 @@ def _divisions(self):
         return self.frame._divisions()
 
 
+class ScalarToSeries(Blockwise):
+    _parameters = ["frame", "index"]
+    _defaults = {"index": 0}
+
+    @staticmethod
+    def operation(value, index=0):
+        return pd.Series(value, index=[index])
```

Comment:
@rjzamora I am merging this to move forward

Thoughts on how you want to handle this for cudf?